File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    add1 = 0
    add2 = 0
    
    if len(arr1) > len(arr2):
        answer = 1
    elif len(arr1) < len(arr2):
        answer = -1
    else:
        for i in arr1:
            add1 += i
        for i in arr2:
            add2 += i
        if add1 > add2:
            answer = 1
        elif add1 < add2:
            answer = -1
        else:
            answer = 0
    return answer

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
import java.util.*;
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int sum1 = Arrays.stream(arr1).sum();
        int sum2 = Arrays.stream(arr2).sum();
        
        if (arr1.length == arr2.length) {
            if (sum1 > sum2)
                answer = 1;
            else if (sum1 < sum2)
                answer = -1;
            else 
                answer = 0;
        } else {
            if (arr1.length > arr2.length)
                answer = 1;
            else 
                answer = -1;
        }
        return answer; 
    }
}